packages/console/src/pages/ApiResourceDetails/ApiResourcePermissions/index.tsx

```diff
@@ -77,20 +80,26 @@ const ApiResourcePermissions = () => {
         errorMessage={error?.body?.message ?? error?.message}
         retryHandler={async () => mutate(undefined, true)}
         pagination={{
-          pageIndex,
+          pageIndex: page,
```

Comment:
should we align the params naming as `pageIndex`?  If it is widely used on all the pages.

Reply:
I renamed `pageIndex` to `page` since `page` is commonly used in URLs and APIs.